go/teams/loader_types.go

```diff
@@ -54,10 +54,13 @@ func unpackChainLink(link *SCChainLink) (*chainLinkUnpacked, error) {
 		inner:       inner,
 		innerLinkID: innerLinkID,
 	}
-
 	return ret, nil
 }
 
+func (l *chainLinkUnpacked) TeamID() (keybase1.TeamID, error) {
```

Comment:
I'm worried someone's going to call this and not realize it only exists if not stubbed.